### ISSUE TITLE
Audit: highlight deprecated pipelines

### ISSUE THREAD

**Issue by sylvain-combe-sonarsource** [okorach/sonar-tools#1609]
Hi @okorach-sonar 
Checking background tasks analysis warnings may allow the tool to highlight deprecated pipeline environments (e.g. sonar.password, Java 11...). That could be quite useful before an upgrade. 
WDYT?

**Comment by okorach** [comment]
Hello @sylvain-combe-sonarsource ,

Assuming that the audit config file has `audit.projects.analysisWarnings = yes`, `sonar-audit` already reports in the audit report any warning from the last background task of any project or branch. See an example below.
It definitely catches usage of `sonar.login`. Anything that is not reported as a warning by the scanner may not be caught for the moment, and I am not sure how the audit could catch that if there is no hint of the problem on the SonarQube Server side.

```
243*****81th,PROJ_ANALYSIS_WARNING,OPERATIONS,MEDIUM,"project 'okorach_sonar-tools' last analysis had the following warnings: The property 'sonar.login' is deprecated and will be removed in the future. Please use the 'sonar.token' property instead when passing a token. --- Missing blame information for 2 files. This may lead to some features not working correctly. Please check the analysis logs and refer to <a href=""https://docs.sonarsource.com/sonarqube/analyzing-source-code/scm-integration/"" rel=""noopener noreferrer"" target=""_blank"">the documentation</a>."
```
